tuxemon/event/conditions/has_party_breeder.py

```diff
@@ -32,8 +33,9 @@ def test(self, session: Session, condition: MapCondition) -> bool:
 
         """
         player = session.player
-        if any(t for t in player.monsters if t.gender == "male"):
-            if any(t for t in player.monsters if t.gender == "female"):
-                return True
+        if any(t for t in player.monsters if t.stage != "basic"):
```

Comment:
What happens if you have an adult male and a basic female?

Reply:
if any(t for t in player.monsters if t.stage != "basic" and t.gender == "male"):
            if any(t for t in player.monsters if t.stage != "basic" and t.gender == "female"):
                    return True

        return False

Reply:
I agree with your suggestion